Example/SnapshotTests/BPKSnapshotTest.h

```diff
@@ -77,3 +77,47 @@
 }
 #endif
 
+#if __BPK_DARK_MODE_SUPPORTED
```

Comment:
Can you give me a little bit of context to understand what these new methods are doing?